gtfs-realtime/proto/gtfs-realtime.proto

```diff
@@ -399,9 +407,56 @@ message VehiclePosition {
   // including both seated and standing capacity, and current operating regulations allow.
   // It is possible that the value goes over 100 if there are currently more passengers than what the vehicle was designed for.
   // The precision of occupancy_percentage should be low enough that you can't track a single person boarding and alighting for privacy reasons.
+  // If multi_carriage_status is populated with per-carriage occupancy_percentage, 
+  // then this field should describe the entire vehicle with all carriages accepting passengers considered.
   // This field is still experimental, and subject to change. It may be formally adopted in the future.
   optional uint32 occupancy_percentage = 10;
 
+  // Carriage specific details, used for vehicles composed of several carriages
```

Comment:
The `CarriageStatus` message and all fields within (except extensions) need the following experimental label, accordingly:

```
// This message/field is still experimental, and subject to change. It may be formally adopted in the future.
```

Reply:
Added.